webpack.prod.js

```diff
@@ -0,0 +1,53 @@
+const webpack = require("webpack");
+const path = require("path");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const merge = require('webpack-merge');
+
+const common = require('./webpack.common.js');
+
+module.exports = merge(common, {
+  mode: "production",
+
+  // devtool: "source-map",
```

Comment:
I'm not sure if we want source maps in production for a components library so I commented it out, but it is a recommended practice. I removed it because it makes the build folder bigger.

Reply:
I'd say leave this in.

Reply:
My only worry is that it creates a map file which is larger than the compiled source.
```
                Asset      Size  Chunks             Chunk Names
    reusable-components.css  2.03 KiB       0  [emitted]  app
reusable-components.css.map  2.81 KiB       0  [emitted]  app
     reusable-components.js   143 KiB       0  [emitted]  app
 reusable-components.js.map   495 KiB       0  [emitted]  app
```

Reply:
I think that's probably ok - the applications using this could choose to ignore those files if they don't need them.